src/cdk/v2/destinations/zoho/types.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  RouterTransformationRequestData,
+  Destination,
+  Connection,
+  Metadata,
+} from '../../../../types';
+
+export type RegionKeys = 'US' | 'AU' | 'EU' | 'IN' | 'CN' | 'JP' | 'CA';
```

Comment:
I think we should declare `RegionKeys` as just `string`. We’ll lose some type specificity, but this way the list won’t get out of sync if Zoho adds new region values. Using a strict enum would be useful if we have logic like switch cases that depended on specific regions.